scheduler/pkg/util/constants.go

```diff
@@ -46,6 +46,21 @@ const (
 	GRPCControlPlaneTimeout      = 1 * time.Minute // For control plane operations except load/unload
 )
 
+// K8s API
+const (
+	K8sTimeoutDefault = 2 * time.Minute
+)
+
+// Servers control plane
+const (
+	ServerControlPlaneTimeout = time.Second * 5
+)
+
+// inference
+const (
+	InferTimeoutDefault = 10 * time.Minute // TODO: expose this as a config (map)?
```

Comment:
In general, I would like us to expose the inference timeout more explicitly and preferably in one place. Atm it's not entirely clear (at least to me) where things timeout (envoy? mlserver? agent? one of the gateways? etc). Perhaps we should add a jira ticket to handle this.